app/javascript/src/shared/styles/mixins/fonts.js

```diff
@@ -4,15 +4,26 @@ type FontFace = {|
   fontWeight?: number | string
 |}
 
-const fontFamily = 'Nunito, sans-serif'
+const fontFamilies = {
+  serif:'Zilla Slab, serif',
+  sans:'Roboto, sans-serif'
+}
 
-export const fonts: { regular: FontFace, bold: FontFace } = {
-  regular: {
-    fontFamily,
+export const fonts: { lightSans: FontFace, regularSans:FontFace, boldSans: FontFace, regularSerif: FontFace } = {
```

Comment:
what do you think about naming these in the format:

`light` for sans styles
`lightSlab` for slab styles

to keep naming terse for the common case

Reply:
sure that sounds good.